src/Components/CountDown/CountDown.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
+import { workTime, shortBreak, longBreak } from "../timeSettings/timeSettings";
+import StartStopButton from "../StartStopButton/StartStopButton";
+import CountdownTimer from "../CountdownTimer/CountdownTimer";
+
+export default function CountDown() {
+  const [timeLeft, setTimeLeft] = useState(workTime);
+  const [isRunning, setIsRunning] = useState(false);
+  const [currentSession, setCurrentSession] = useState("workTime");
+  const [currentPeriod, setCurrentPeriod] = useState(0);
+
+  const startHandler = () => {
+    setIsRunning(true);
+  };
+
+  const stopHandler = () => {
+    setIsRunning(false);
+  };
+
+  const switchToNextSession = () => {
+    if (currentSession === "shortBreak") {
+      setCurrentSession("workTime");
+      setTimeLeft(workTime);
+    } else if (currentSession === "workTime") {
+      setCurrentSession("shortBreak");
+      setTimeLeft(shortBreak);
+      setCurrentPeriod(count => count + 1);
+    } else if (currentSession === "longBreak") {
+      setCurrentSession("workTime");
+      setTimeLeft(workTime);
+      setCurrentPeriod(count => count + 1);
+    }
+  };
```

Comment:
Can you move this function into a separate file, and for example you can name it timerHelper.js